opensrp-app/src/main/java/org/smartregister/util/DatabaseMigrationUtils.java

```diff
@@ -156,6 +156,32 @@ public static void recreateSyncTableWithExistingColumnsOnly(SQLiteDatabase datab
     }
 
 
+    public static void recreateSyncTableWithExistingColumnsOnly(SQLiteDatabase database, String table ,String[] columns, String createTableDDL) {
+        database.beginTransaction();
+        //rename original table
+        database.execSQL("ALTER TABLE " + table + " RENAME TO " + TABLE_PREFIX + table);
+        //recreate table
+        database.execSQL(createTableDDL);
+        //
+        String insertQuery = "INSERT INTO "
+                + table
+                + " (" + StringUtils.join(table, ", ") + ")"
```

Comment:
should be `columns` and not `table` here

Reply:
nice catch, updated